MobileClient/Client/Pages/OrderDetails.razor

```diff
@@ -0,0 +1,50 @@
+﻿@page "/Orders/{Id:long}"
+
+@inherits BasePage
+
+<AuthorizeView>
+    <NotAuthorized>
+        <RedirectToLogin/>
+    </NotAuthorized>
+    <Authorized>
+        <ConditionComponent Condition="!IsLoading">
+            <Match>
+                <ConditionComponent Condition="order != null">
+                    <Match>
+                        <div class="mainContent">
+                            <MudText Align="Align.Center">Order from @order.Date</MudText>
+                            <MudDivider Class="mb-3 mt-2" Style="border-color:#808080; border-width:0.2rem;" />
+                            <MudText Align="Align.Center" Typo="Typo.h5">Thank you for your order, your games are activated</MudText>
```

Comment:
Thank you for your purchase!

Das anderer würde ich da glaube weg lasse. Ist ja eigentlich selbstverständlich.